src/Symfony/Component/Console/Helper/TableStyle.php

```diff
@@ -132,6 +139,127 @@ public function getCrossingChar()
         return $this->crossingChar;
```

Comment:
you should deprecate `getCrossingChar()` and `setCrossingChar()`

Reply:
I deprecated `setCrossingChar()`. But not the getter which is always used as-is.

Reply:
That's the point of a deprecation. Update the calls and trigger a deprecation message. :}